blog-posts/step-function-learnings/step-function-logs-learning.md

```diff
@@ -0,0 +1,88 @@
+
+In the following images, you can see that for three different State Machines one Express configured with logs, one Express configured without logs and one Standard, the only one that has a log group associated is the Express one configured with a log group.
+
+![Three types of state machine](./assets/three-types-of-state-machine.png 'Three types of state machine')
+
+![One log group associated to one of the Express State machine](./assets/one-log-group.png 'One log group associated to one of the Express State machine')
+
+And the logs available for the Standard type of State Machine are inside the AWS Step Function service but there is no log group associated.
+
+![Logs of Standard State Machine is in the AWS Step Function service](./assets/one-log-group.png 'Logs of Standard State Machine is in the AWS Step Function service')
+
+## CDK is your best friend
+
+![CDK is da real MVP](./assets/youDaRealMVP.jpeg 'CDK is da real MVP')
+
+All you need is to create a log group, configure it and then attach it to the State Machine via the logs prop. If you want to jump directly to the snippet [here](#how-to-configure-logs-in-an-express-step-function-in-cdk) is a shortcut.
+
+The cdk handles all the work of creating the role for the step function to write inside CloudWatch.
+
+The LogLevel allows you to select what kind of information you want to have. The four levels are: `OFF`, `ALL`, `ERROR` and `FATAL`.
+
+But the `includeExecutionData` property of the CDK construct makes all the difference. Thanks to this property set to `true`, you have all step transition data displayed in the Step Function console.
+
+|                                                                                               Express with NO *logs execution data*                                                                                                |                      Express with *logs execution data* because `includeExecutionData` is set to `true`                       |
+| :--------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------: | :---------------------------------------------------------------------------------------------------------------------------: |
+| ![Express with logs execution data NOT logged because no includeExecutionData](./assets/express-with-logs-but-not-includeExecutionData-exe-logs.png 'Express with logs execution data NOT logged because no includeExecutionData') | ![Express with logs execution data logged](./assets/express-with-logs-exe-logs.png 'Express with logs execution data logged') |
+
+### How to configure logs in an Express Step Function in CDK
+
+This is pretty easy and straightforward but you just need to know it.
```

Comment:
pas sûr de la valeur qu'apporte cette phrase. Pour des gens qui découvrent le CDK c'est pas forcément easy, et ça peut passer pour un énorme 🍈 de ta part. Quid de 

"Notice the ìncludeExecutionLogs` property"